src/index.html

```diff
@@ -5,8 +5,483 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="Logo Moyo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="list header__nav-list">
+          <li class="list__item">
+            <a href="#" class="list__item-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a
+              class="list__item-link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="list__item">
+            <a href="#" class="list__item-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main class="main">
+
+      <div class="main__cards-first_row">
```

Comment:
```suggestion
    <main class="main">
      <div class="main__cards-first_row">
```
Don't leave newlines between parent and child elements